cmd/controller/main.go

```diff
@@ -75,6 +85,20 @@ func main() {
 
 	csi.RegisterIdentityServer(csiControllerServer.GRPCServer, controllerService)
 	csi.RegisterControllerServer(csiControllerServer.GRPCServer, controllerService)
+
+	grpc_prometheus.Register(csiControllerServer.GRPCServer)
+	grpc_prometheus.EnableHandlingTimeHistogram()
```

Comment:
Why does it isn't inside in `if enableMetrics {` block?

Reply:
+1, but not critical.

Reply:
Fixed